File: algorithm/typescript/BOJ2739.ts
// problem : https://www.acmicpc.net/problem/2739

import fs = require("fs");

const N: number = Number(fs.readFileSync("/dev/stdin").toString());
const SIZE: number = 9;
const result: string[] = new Array();

for (let i = 1; i <= SIZE; i++) {
  result.push(`${N} * ${i} = ${N * i}`);
}

console.log(result.join("\n"));

File: algorithm/java/src/BOJ2739.java
// problem : https://www.acmicpc.net/problem/2739

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class BOJ2739 {
    static int N;
    static int SIZE = 9;

    public static void main(String[] args) throws Exception {
        StringBuilder sb = new StringBuilder();
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        N = Integer.parseInt(br.readLine());

        for (int i = 1; i <= SIZE; i++) {
            sb.append(N).append(" * ").append(i).append(" = ").append(N * i).append("\n");
        }

        sb.setLength(sb.length() - 1);
        System.out.println(sb);
    }
}
